jaxrs-clients/src/main/java/com/palantir/remoting2/jaxrs/feignimpl/FeignSerializableErrorErrorDecoder.java

```diff
@@ -45,7 +45,6 @@ public Exception decode(String methodKey, Response response) {
         } catch (IOException e) {
             return new RuntimeException("Cannot get input stream from response: " + e.getMessage(), e);
         }
-        return SerializableErrorToExceptionConverter.getException(contentTypes, response.status(), response.reason(),
-                body);
+        return SerializableErrorToExceptionConverter.getException(contentTypes, response.status(), body);
```

Comment:
why stop passing through the reason? seems useful as a more human-readable version of the status code.